tests/docker_service_fixtures.py

```diff
@@ -37,13 +37,13 @@ async def wait_until_responsive(
     """
     ref = timeit.default_timer()
     now = ref
-    while (now - ref) < timeout:
+    while now - now < timeout:
```

Comment:
This is a bug and seems to have broken our tests.

Reply:
fixed

Reply:
thats not the reason for the failure, although i reverted this change. The reason is that the event loop closes for some reason.